arm/monitor/servicediagnosticsettings.go

```diff
@@ -1,242 +0,0 @@
-package monitor
```

Comment:
Is it ok that this file is deleted?

Reply:
I think this file was renamed yes?

Reply:
Still looking

Reply:
Looks like operations moved to different operation group